### ISSUE TITLE
Updated developer experience

### ISSUE THREAD

**Issue by SysC0mp** [MicrosoftDocs/dynamics-365-unified-operations-public#2367]
Any news on the new installable developer tools? It was initially planned for October 2019.
see: https://powerusers.microsoft.com/t5/Microsoft-Business-Applications/Microsoft-Dynamics-365-for-Finance-and-Operations-Updated/m-p/299814

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2876dff1-4dae-cdba-09a3-68abfd509f3f
* Version Independent ID: ffcfca9b-ab9a-54ce-2d80-1d17654ec49b
* Content: [Platform updates for version 10.0.11 of Finance and Operations apps (July 2020) - Finance & Operations | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/get-started/whats-new-platform-update-35)
* Content Source: [articles/fin-ops-core/dev-itpro/get-started/whats-new-platform-update-35.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/fin-ops-core/dev-itpro/get-started/whats-new-platform-update-35.md)
* Service: **dynamics-ax-platform**
* Product: ****
* Technology: ****
* GitHub Login: @sericks007
* Microsoft Alias: **sericks**

**Comment by tonyafehr** [comment]
For the latest details about planned features, see the [Release plans](https://docs.microsoft.com/dynamics365-release-plan/2020wave1/).

**Comment by SysC0mp** [comment]
I am reading that constantly but I cannot find any information about that anywhere. It was simply removed and nobody can give any information about it.

**Comment by jorisdg** [comment]
You probably have noticed that we kept moving the date out due to more work and unexpected delays. Since this was counter-productive to the goal of the release notes, we had to take it off entirely until we are comfortable about sharing a date we can realistically make. This feature is still actively being worked on, so it's definitely still planned (just not publicly). Expect more news around the summer on what we're doing.
